src/model_converter/shape_parser.py

```diff
@@ -0,0 +1,30 @@
+import os
+import yaml
+
+
+def make_string_of_one_shape_with_new_batch(original_shape, new_batch):
+    dims = original_shape[1:]
+    res = f'[{new_batch}'
+    for dim in dims:
+        res += f',{dim}'
+    res += ']'
```

Comment:
```suggestion
    dims.insert(0,new_batch)
    res = str(dims).replace(' ', '')
```

Возможно это дело вкуса :blush:

Reply:
@FenixFly, вариант Александры выглядит более читаемым.

Reply:
исправлено